samples/sample-rdk-rules/lambda_helper.py

```diff
@@ -0,0 +1,141 @@
+import argparse
+import json
+import os
+import shutil
+import uuid
+import sys
+import logging
+import boto3
+from s3 import S3
+from pathlib import Path
+
+
+deployment_account_region = os.environ.get("AWS_REGION")
+project_root = os.path.dirname(__file__)
+s3_rdk_assets_prefix = "rdk_assets"
+config_rules_dir = "config-rules"
+templates_dir = "templates"
+config_rules_root = os.path.join(project_root, "config-rules")
+templates_root = os.path.join(project_root, templates_dir)
+
+def load_json_file(file: str) -> dict:
+    try:
+        with open(f"{file}") as file:
```

Comment:
When you open a file, it is best to tell it the encoding if it is a text file like JSON. Like so:
```suggestion
        with open(f"{file}", "r", ecoding="utf-8") as file:
```

See https://pylint.pycqa.org/en/latest/user_guide/messages/warning/unspecified-encoding.html

Reply:
Fixed